README.md

```diff
@@ -73,7 +161,7 @@ ReactDOM.render(
       header={<Cell>Col 3</Cell>}
       cell={({rowIndex, ...props}) => (
         <Cell {...props}>
-          Data for column 3: {rows[rowIndex][2]}
+          Data for column 3: {rows[rowIndex]}
```

Comment:
If we do this, could you change rows to just be an array of values instead of array of arrays